frontend/src/index.css

```diff
@@ -7,6 +7,15 @@ body {
     -moz-osx-font-smoothing: grayscale;
 }
 
+button {
+    color: white;
+    background-color: #1976D2;
```

Comment:
As mentioned in another File: We plan to introduce a theming api later, by hardcoding this here we are running into problems later.